packages/indicators/src/builders/buildArithmetic/config/types.ts

```diff
@@ -0,0 +1,38 @@
+/**
+ * Tupaia
+ * Copyright (c) 2017 - 2020 Beyond Essential Systems Pty Ltd
+ */
+
+import { Aggregation, Indicator } from '../../../types';
+
+/**
+ * A single aggregation
+ */
+export type AggregationDescriptor = string | Aggregation;
+
+/**
+ * A list of aggregation(s) that will be applied to a data element
+ */
+export type AggregationSpecs = AggregationDescriptor | AggregationDescriptor[];
+
+/**
+ * A flexible config schema that can be provided as an input to the indicator
+ */
+export type ArithmeticConfig = {
+  readonly formula: string;
+  // If `AggregationSpecs` (and not a Record) is passed, it will be applied to all elements in `formula`
+  readonly aggregation: AggregationSpecs | Record<string, AggregationSpecs>;
+  readonly parameters?: Indicator[];
+  readonly defaultValues?: Record<string, number>;
+};
+
+/**
+ * A fully expanded, verbose config containing as much information as possible.
+ * Useful for internal code purposes
+ */
+export type ExpandedArithmeticConfig = {
```

Comment:
`Expanded` is probably fine but I'd be happy to consider other alternatives (`Verbose`? `Full`?)

I don't know if there is a standard word for this behaviour. I also considered `Sanitised`, but I think it mainly describes the removal of dangerous/unnecessary data

Reply:
I'm fine with `Expanded` I think it works well